src/workerd/api/queue.h

```diff
@@ -63,9 +71,198 @@ class WorkerQueue: public jsg::Object {
   uint subrequestChannel;
 };
 
+// Event handler types
+
+// Types for other workers passing messages into and responses out of a queue handler.
+
+struct IncomingQueueMessage {
+	kj::String id;
+	kj::Date timestamp;
+	kj::Array<kj::byte> body;
+	JSG_STRUCT(id, timestamp, body);
```

Comment:
Nit: Tabs here and below instead of spaces? FWIW, I'd be fine with switching to tabs in general but for consistency with everywhere else :-)

Reply:
Weird, I have no idea how these got here since my editor uses spaces rather than tabs. Thanks for pointing out.